examples/LED/LED.ino

```diff
@@ -52,9 +63,21 @@ void setup(void){
   adapter.addDevice(&led);
   adapter.begin();
   Serial.println("HTTP server started");
+  Serial.print("http://");
+  Serial.print(WiFi.localIP());
+  Serial.print("/things/");
+  Serial.println(led.id);
 }
 
 void loop(void){
   adapter.update();
-  digitalWrite(ledPin, ledOn.getValue().boolean ? LOW : HIGH); // active low led
+  status = ledOn.getValue().boolean;
+  digitalWrite(ledPin, status ? LOW : HIGH); // active low led
+  if (before != status) {
```

Comment:
Since `before` is only used to keep track of the last value of the led's `on` property, I think `lastOn` would be a clearer name for it.